caller: add page (#15246)
tldr-pages/tldr: pages/common/caller.md

# caller

> Print function context.
> More information: <https://www.gnu.org/software/bash/manual/bash.html#index-caller>.

- Print the line where the current function was called and its filename:

`caller`

- Print the line where the current function was called, its function name and filename:

`caller 0`

> - Print the line where a frame above was called, its function name and filename:
> 
> `caller {{1}}`

Reviewer: > I need help with this page as I'm not sure if the language used here is entirely correct. I might be using words like "frame" wrong.

Maybe like this?

```suggestion
- Print the line starting the n'th-last function call, its function name and filename:

`caller {{n}}`
```

Author: I guess using technical words like frames won't help newbies either. I'll take that suggestion and massage it a bit.

Author: I think I got the wording in a good position now.

Reviewer: agree :+1:
